### ISSUE TITLE
deleting the saved and intermediate folders reset the config files

### ISSUE THREAD

**Issue by IronSuit** [alexhajdu/fix_this#10]
Hello, I always delete the saved and intermediate folders when i backup my project, however i noticed that the config panel get reset to initial setting when i do so.

**Comment by alexhajdu** [comment]
Yes, this will be solved in the upcoming version with the mentioned way.

**Comment by IronSuit** [comment]
I really appreciate you support, Many Thanks.

**Comment by IronSuit** [comment]
Has this be solved yet?

**Comment by alexhajdu** [comment]
> Has this be solved yet?

It's ready and it will be shipped in the next major update.
